test/d2l-user-card-auto/d2l-user-card-auto.js

```diff
@@ -1,101 +1,98 @@
-/* global describe, beforeEach, afterEach, it, expect, fixture, sinon, Promise */
-
 'use strict';
 import { afterNextRender } from '@polymer/polymer/lib/utils/render-status';
 
-describe('<d2l-user-card-auto>', function() {
-	var component,
-		sandbox,
-		userUrl = 'https://example.com',
-		token = 'some-oauth-token',
-		getToken = function() {
-			return Promise.resolve(token);
-		},
-		dontGetToken = function() {
-			return Promise.reject(new Error('No token for that child'));
-		},
-		getNotToken = function() {
-			return Promise.resolve(null);
-		};
+describe('<d2l-user-card-auto>', () => {
+	let component, sandbox;
+	const userUrl = 'https://example.com';
+	const token = 'some-oauth-token';
+	const getToken = function() {
+		return Promise.resolve(token);
+	};
+	const dontGetToken = function() {
+		return Promise.reject(new Error('No token for that child'));
+	};
+	const getNotToken = function() {
+		return Promise.resolve(null);
+	};
```

Comment:
Nit, just for consistency
```suggestion
	const getToken = () => {
		return Promise.resolve(token);
	};
	const dontGetToken = () => {
		return Promise.reject(new Error('No token for that child'));
	};
	const getNotToken = () => {
		return Promise.resolve(null);
	};
```

Reply:
Good point, I was trying not to touch the test files (they're old polymer and can be a bit finicky) but this still works, so 🚢